VimeoNetworking.playground/Sources/VimeoClientFactory.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import VimeoNetworking
+
+private let clientID = ""
+private let clientSecret = ""
+private let scopes: [Scope] = [.Public, .Private, .VideoFiles]
+
+public let token = ""
```

Comment:
At first I was thinking it might be cool to create some Xcode placeholders for the empty strings…

<img width="475" alt="Screen Shot 2019-10-15 at 1 31 17 PM" src="https://user-images.githubusercontent.com/19800/66854854-26ddc100-ef50-11e9-908f-f26904995aff.png">

… but that won't compile and you'll just get errors, so that's not great.

Our documentation should explain how to use these things, but it still might be helpful to add a comment  to indicate that these have to be obtained elsewhere. I'm thinking somebody coming into the playground for the first time might overlook it since we're down here in the `Sources` directory.